Readme.md

```diff
@@ -4,11 +4,16 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
-# How to use an unbound check box column to select grid rows
+# Data Grid for Windows Forms - How to use an unbound check box column to select grid rows
 
+The [Grid View](https://docs.devexpress.com/WindowsForms/3464/controls-and-libraries/data-grid/views/grid-view) in the [Data Grid](https://docs.devexpress.com/WindowsForms/3455/controls-and-libraries/data-grid) control supports a built-in checkbox column for multiple row selection. 
```

Comment:
On line 7 above you mention "check box", but on this line you say "checkbox". Can you please make it consistent?